.github/workflows/build_and_test_workflow.yml

```diff
@@ -206,7 +206,7 @@ jobs:
 
       # image has the latest chrome v99
       - name: Setup chromedriver
-        run: yarn add --dev chromedriver@106.0.1
```

Comment:
the github public image uses chrome 99?

Reply:
I don't know - but that's what line 157 says...